openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/WheelchairGraphStorageBuilder.java

```diff
@@ -313,8 +318,8 @@ private void setSidewalkAttributeForSide(String value, WheelchairAttributes.Attr
 	 * @param value			The value to store
 	 * @param attribute		The attribute to store the value against
 	 */
-	private void setWheelchairAttribute(String value, WheelchairAttributes.Attribute attribute) {
-		wheelchairAttributes.setAttribute(attribute, convertTagValueToEncodedValue(attribute, value));
+	private void setWheelchairAttribute(String value, WheelchairAttributes.Attribute attribute, boolean markSurfaceQualityKnown) {
```

Comment:
Should add the new attribute in the JDoc items

Reply:
done